src/rust/bitbox02/src/ui/ui_stub.rs

```diff
@@ -75,6 +75,13 @@ where
     panic!("not implemented")
 }
 
+pub fn sdcard_create<'a, F>(_insert: bool, _continue_callback: F) -> Component<'a>
+where
+    F: FnMut() + 'a,
+{
+    panic!("not implemented")
```

Comment:
Forgot: wouldn've been also nice to make a test with this.

Reply:
done, PTAL

Reply:
:+1: thanks!